frontend/src/entry/settings/EntryAccessInput.tsx

```diff
@@ -72,6 +73,18 @@ export default function EntryAccessInput({ value, onUpdate, entryKind, entryRole
     setUserListQuery(value)
   }, [])
 
+  const handleNewManualEntityEntry = useCallback(
```

Comment:
This doesn't need to be using `useCallback`.

It also should have a name that matches the event it's linked to: `handleAddEntityManually`.

Side note, we should be careful about our use of the word `entry` as it has a specific meaning in the codebase.